newsfragments/41039.feature.rst

```diff
@@ -0,0 +1 @@
+Enable ``get_current_context()`` to work in virtual environments. This is a dictionary object, not a model defined by Airflow, so care should be taken when using it.
```

Comment:
Lets be explicit about the relevant operators that this applies to (or the operator family)
This is important as when user ia reading the changelog they need to know what each entry is about

Reply:
Due to changes to the deserialization method, I need to change the whole thing, I'll add it then.

Reply:
Updated the documentation.